### PR TITLE
docs: added examples

### PR DESCRIPTION
**Pull request by mikekenneth**
fixes https://github.com/domoritz/csv2parquet/issues/84

Added the below example:
- Convert a CSV to PARQUET
- Convert a CSV with no `header` to PARQUET
- Get the `schema` from a CSV with header
- Convert a CSV using `schema-file` to PARQUET

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -88,3 +88,46 @@ OPTIONS:
 ```
 
 The --schema-file option uses the same file format as --dry and --print-schema.
+
+## Examples
+### Convert a CSV to PARQUET
```
Comment (domoritz) [inline Readme.md:93]: Parquet isn't an abbreviation so let's spell it differently

```suggestion
### Convert a CSV to Parquet
```

File: Readme.md [hunk 2]
```diff
@@ -88,3 +88,46 @@ OPTIONS:
 ```
 
 The --schema-file option uses the same file format as --dry and --print-schema.
+
+## Examples
```
Comment (domoritz) [inline Readme.md:92]: Add empty lines after headings

```suggestion
## Examples

```

File: Readme.md [hunk 3]
```diff
@@ -88,3 +88,46 @@ OPTIONS:
...
+csv2parquet --header false <CSV> <PARQUET>
+```
+
+### Get the `schema` from a CSV with header
+```bash
+csv2parquet --header true --dry <CSV> <PARQUET>
+```
+
+### Convert a CSV using `schema-file` to PARQUET
+- Below is an example of the `schema-file` content:
```
Comment (domoritz) [inline Readme.md:109]:
```suggestion

Below is an example of the `schema-file` content:

```

File: Readme.md [hunk 4]
```diff
@@ -88,3 +88,46 @@ OPTIONS:
...
+      "name": " col2",
+      "data_type": "Utf8",
+      "nullable": false,
+      "dict_id": 0,
+      "dict_is_ordered": false
+    }
+  ]
+}
+```
+- Then add the schema-file `schema.json` in the command:
```
Comment (domoritz) [inline Readme.md:130]:
```suggestion

Then add the schema-file `schema.json` in the command:
```

**Review by domoritz — CHANGES_REQUESTED** [review]
Thank you. Just added a few suggestions.

**Comment by mikekenneth** [comment]
> Thank you. Just added a few suggestions.

Added those suggestions. Thanks.

**Pull request merged**